google/cloud/storage/_http.py

```diff
@@ -65,14 +66,25 @@ def __init__(self, client, client_info=None, api_endpoint=None):
 
     def api_request(self, *args, **kwargs):
         retry = kwargs.pop("retry", None)
-        kwargs["extra_api_info"] = _helpers._get_invocation_id()
+        invocation_id = _helpers._get_invocation_id()
+        kwargs["extra_api_info"] = invocation_id
+        span_attributes = {
+            "gccl-invocation-id": invocation_id,
+        }
         call = functools.partial(super(Connection, self).api_request, *args, **kwargs)
-        if retry:
-            # If this is a ConditionalRetryPolicy, check conditions.
-            try:
-                retry = retry.get_retry_policy_if_conditions_met(**kwargs)
-            except AttributeError:  # This is not a ConditionalRetryPolicy.
-                pass
+        with create_span(
+            name="Storage.Connection.api_request",
```

Comment:
How did you arrive at this name layout?

Reply:
This follows the span name convention of ​​$package.$service/$method. Basically, the span name should identify the work represented by the span, in this case, it should include the RPC method name. Internal go/gcs-sdk-tracing-span-attributes is tracking the semantic conventions for spans.